charts/eol-exporter/Chart.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v2
+name: eol-exporter
+description: |
+  End of life exporter.
+  A Kubernetes's helm chart for a exporter that get information about end of life/support of products in order to be scrapped by Prometheus
```

Comment:
provide a lot more details how to perform configuration, options for configmap etc.

Reply:
Added